apps/platform/src/components/integrations/integrationServices/index.tsx

```diff
@@ -0,0 +1,66 @@
+'use client'
+import React, { useCallback, useMemo } from 'react'
+import { Integrations } from '@keyshade/common'
+import { useAtom, useSetAtom } from 'jotai'
+import type { Integration } from '@keyshade/schema'
+import IntegrationIcon from '../integrationIcon'
+import CreateIntegration from '../createIntegration'
+import { Button } from '@/components/ui/button'
+import { createIntegrationOpenAtom, createIntegrationTypeAtom } from '@/store'
+
+export default function IntegrationServices(): React.JSX.Element {
+  const integrations = useMemo(() => Object.values(Integrations), [])
```

Comment:
fix `Unsafe argument of type `any` assigned to a parameter of type `{ [s: string]: unknown; } | ArrayLike<unknown>``